From 01d53ae63b68f2279d3a11a59df6811bcccc33ff Mon Sep 17 00:00:00 2001
Message-Id: <patch-4.4-01d53ae63b6-20210928T125228Z-avarab@gmail.com>
In-Reply-To: <cover-0.4-00000000000-20210928T125228Z-avarab@gmail.com>
References: <cover-0.4-00000000000-20210928T125228Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 27 Sep 2021 01:33:33 +0200
Subject: [PATCH 4/4] config.h: remove unused git_config_get_untracked_cache()
 declaration
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This function was removed in ad0fb659993 (repo-settings: parse
core.untrackedCache, 2019-08-13), but not its corresponding *.h entry.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 config.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/config.h b/config.h
index 147f5e0490e..7378fc165fb 100644
--- a/config.h
+++ b/config.h
@@ -609,7 +609,6 @@ int git_config_get_maybe_bool(const char *key, int *dest);
 int git_config_get_pathname(const char *key, const char **dest);
 
 int git_config_get_index_threads(int *dest);
-int git_config_get_untracked_cache(void);
 int git_config_get_split_index(void);
 int git_config_get_max_percent_split_change(void);
 int git_config_get_fsmonitor(void);
-- 
2.33.0.1340.ge9f77250f2b

